lib/Button/styles.scss

```diff
@@ -89,9 +89,19 @@ $color-btn-default-dark--hover: #5e5e5e !default;
 }
 
 .button--link-default {
-  color: $color-btn-secondary--hover;
+  color: $color-brand-blue;
```

Comment:
`button--link` doesn't give us a proper link styling but seeing as its used else where I have created a `link-default` modifier to get the blue text colour.

We also needed a link button modifier for danger links too.